mock_sequence.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022 ByteDance Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package mockey
+
+import "github.com/bytedance/mockey/internal/tool"
+
+type SequenceOpt interface {
```

Comment:
why `SequeceOpt` and `sequenceOpt` are all interfaces?

Reply:
Provide an interface that users can implement it by themselves